src/Mapping/Driver/MappingTrait.php

```diff
@@ -97,7 +97,7 @@ protected function getGroupMetadata(array $mapping, GroupMetadata $parentGroup =
      *
      * @return RouteMetadata
      */
-    protected function getRouteMetadata(array $mapping, GroupMetadata $group = null): RouteMetadata
+    protected function getRouteMetadata($mapping, GroupMetadata $group = null): RouteMetadata
```

Comment:
Why remove this typehint? $mapping is still an array

Reply:
I accidentally removed it while I was trying to get it working, I guess. I added it back in!